### DISCUSSION TITLE
Is the project abandoned?

### DISCUSSION THREAD

**Discussion by Insight16** [AdguardTeam/AdGuardHome#7299] [General]
Hello @ainar-g! There have been no updates/fixes for a long time. Everything seems to work, but still. Are you no longer developing this project? :,(

**Comment by ppfeufer** [comment]
![2024-09-30_08-29-35](https://github.com/user-attachments/assets/0f19d5e2-1c7d-48db-9d50-678d47b69505)
I wouldn't call this abandoned at all. The latest release was just 4 months ago.

**Reply by Insight16** [reply]
I also cannot call it completely abandoned, but I judge only by its latest activity. I also think that the people who led this project have changed.

**Reply by Insight16** [reply]
Even in the changelog (raw) it still says the month of the new version release is July 2024
![123](https://github.com/user-attachments/assets/a6dda835-67a4-444b-9429-5a952b72040f)

**Comment by Dynamic5912** [comment]
No fixes is sometimes a good thing though. It means everything is working as it should be.

I'm pretty sure given the userbase that they would announce publicly if this was indeed "abandoned".

**Reply by Insight16** [reply]
I hope everything is fine. The project is very cool, I've been using it for a long time. I'm just worried...

**Comment by MatiDEV-PL** [comment]
Just now, the beta version was released v0.108.0-b.58, so it's safe to assume it's not abandoned.

**Reply by Insight16** [reply]
Yeah, that's cool!

**Comment by Insight16** [comment]
Dear @ainar-g! Thank you and your entire team for your work, for your labor. To be honest, I did not assume that there are issues, the solution of which will take quite a long time. I thought that "already everything". It's good that I was wrong. Thank you again! You are very cool! :)
